### PR TITLE
Update BarVaryingDistributedLoad

### PR DESCRIPTION
**Pull request by IsakNaslundBh**
### NOTE: Depends on 

https://github.com/BHoM/BHoM/pull/1075
https://github.com/BHoM/BHoM_Engine/pull/2160
   
### Issues addressed by this PR

Aligns with changes to the BarVaryingDistributedLoad.

### Test files

https://burohappold.sharepoint.com/:f:/s/BHoM/EhIZk-booZNHhaTeGJLSuMUB0TS-q8UA6_WxZH8crV4fWg?e=WkN1Ni

### CODE REVIEW

File: Robot_Adapter/Convert/ToRobot/Loads/BarVaryingDistributedLoad.cs [hunk 1]
```diff
@@ -36,51 +36,42 @@ public static partial class Convert
...
 
-            if (load.ForceA.Length() == 0 && load.ForceB.Length() == 0)
+            if (load.ForceAtStart.Length() == 0 && load.ForceAtEnd.Length() == 0)
             {
                 Engine.Reflection.Compute.RecordWarning("Zero distributed forces are not pushed to Robot");
                 return;
             }
 
-            if (load.MomentA.Length() != 0 && load.MomentB.Length() != 0)
+            if (load.MomentAtStart.Length() != 0 && load.MomentAtEnd.Length() != 0)
```
Comment (peterjamesnugent) [inline Robot_Adapter/Convert/ToRobot/Loads/BarVaryingDistributedLoad.cs:50]:
```suggestion
            if (load.MomentAtStart.Length() != 0 || load.MomentAtEnd.Length() != 0)
```

You could have the case where the moment goes from Mx =0 kNm to Mx = +5 kNm

**Review by peterjamesnugent — CHANGES_REQUESTED** [review]
Works well for me, I've updated the link to the test script (isolated `BarVaryingDistributedLoad` objects and added their new parameters and a few edge cases).

One tweak made so that users are aware that Moments are not supported in Robot.

**Review by peterjamesnugent — APPROVED** [review]
LGTM.
